app/models/account/states.rb

```diff
@@ -77,7 +79,7 @@ module Account::States
       end
 
       event :suspend do
-        transition :approved => :suspended, if: :provider_but_not_master?
+        transition all => :suspended, if: :provider_but_not_master?
```

Comment:
Same as https://github.com/3scale/porta/pull/17/files#r229268488. Can it be `tenant?`?

Reply:
I didn't create that method... so that sounds to me like out of the scope of this PR 😛

Reply:
i don't have any problem with `provider_but_not_master?`

Reply:
It should be an alias :)

Reply:
An alias only for this method? Imo we should have only one name for this method and no alias 😕 It was created in a very recent PR and it doesn't have many references to it, so I don't see why in the next PR to use it we go and create an alias! 😱

Reply:
It may not belong to this PR individually, but the change has some context with the new scope `tenants` you introduce here. Besides, it's that many occurrences anyway:
https://github.com/3scale/porta/blob/9402f1f555910c1969ea0edbdeecea3afb66bbdd/app/models/account.rb#L383
https://github.com/3scale/porta/blob/9402f1f555910c1969ea0edbdeecea3afb66bbdd/app/models/account/states.rb#L80
https://github.com/3scale/porta/blob/9402f1f555910c1969ea0edbdeecea3afb66bbdd/test/unit/account_test.rb#L35-L44

Reply:
Ok, I will change it in another independent PR first then 👍 And I will want you all to review it there and speak about it there 😝

Reply:
Moved to https://github.com/3scale/porta/pull/269 😄